packages/app/lib/entrypoints/android/main_huawei_appgallery.dart

```diff
@@ -1,8 +1,12 @@
+import 'package:scanner_mlkit/scanner_mlkit.dart';
 import 'package:smooth_app/main.dart';
 
 /// Huawei App Gallery version with:
 /// - Barcode decoding algorithm: ZXing
 /// - Intent to launch the review
 void main() {
-  launchSmoothApp();
+  launchSmoothApp(
+    // TODO(g123k): Replace this when ZXing is ready
```

Comment:
The Huawei store doesn't exclude ML Kit does it

Reply:
On Amazon devices, they don't have the Google Play Services

Reply:
Yeah of course, but as far as I know is Huawai not on the blacklist anymore and because we use the bundled version we don't need Google Play services

Reply:
This is the same as the Amazon App Store. Some devices may miss some APIs, so it's simpler to only have the ZXing version here

Reply:
Fair enouph